src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx

```diff
@@ -5,10 +5,26 @@ import { AuctionArtworksRail_sale } from "v2/__generated__/AuctionArtworksRail_s
 import { RouterLink } from "v2/Artsy/Router/RouterLink"
 import { useLazyLoadComponent } from "v2/Utils/Hooks/useLazyLoadComponent"
 import { AuctionArtworksRailArtworksQueryRenderer } from "./AuctionArtworksRailArtworks"
-import { AuctionArtworksRailPlaceholder } from "./AuctionArtworksRailPlaceholder"
+import { AuctionArtworksRailPlaceholder } from "../AuctionArtworksRailPlaceholder"
+import { getContextModule } from "../../Utils/getContextModule"
+import { useTracking } from "react-tracking"
+import {
+  clickedArtworkGroupHeader,
+  ClickedArtworkGroupHeaderArgs,
+  OwnerType,
+} from "@artsy/cohesion"
```

Comment:
Cohesion is where we store our analytics schema: https://github.com/artsy/cohesion. When you get a sec give the readme and [the docs](https://cohesion.artsy.net/) a good read just to get kinda familiar.